src/pivotal-ui/components/forms/forms.scss

```diff
@@ -1549,6 +1551,8 @@ the way we see fit.
 ```
 */
 
+$form-input-error-height: 35px;
```

Comment:
Honestly? Working with jerry on making errors show nicely. Kept playing with it until we hit 35px